src/opendr/engine/target.py

```diff
@@ -758,3 +758,30 @@ def __str__(self):
             return f"Class {self.data} speech command with confidence {self.confidence}"
         else:
             return f"Class {self.data} speech command"
+
+
+class Heatmap(Target):
```

Comment:
Please also add a short description in https://github.com/tasostefas/opendr_internal/blob/master/docs/reference/engine-target.md